src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonPropertyInfo.cs

```diff
@@ -824,6 +824,6 @@ internal JsonTypeInfo JsonTypeInfo
         internal abstract object? DefaultValue { get; }
 
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        private string DebuggerDisplay => $"MemberInfo={AttributeProvider as MemberInfo}";
+        private string DebuggerDisplay => $"DeclaringType = {DeclaringType}, PropertyType = {PropertyType}, Name = {Name}";
```

Comment:
I'll suggest to put name first, then property type as it will likely make debugging easier

Reply:
FWIW `PropertyInfo` uses a `{PropertyType} {Name}` format. Maybe we should just use that and skip the declaring type?